lib/index.es6

```diff
@@ -0,0 +1,34 @@
+import path from "path";
+import fs from "fs";
+
+import {sync as mkdirp} from "mkdirp";
+
+import compile from "./compile";
+import {entries} from "./util";
+
+export default function Interlock (options) {
+  const cwd = process.cwd();
+  options = options || {};
+
+  if (!options.emit || !options.emit.length) {
+    throw new Error("Must define at least one bundle.");
+  }
+
+  // TODO: validate options, or only persist specific values
```

Comment:
Somebody just turned me on to [validate.js](http://validatejs.org/) for object validation. Looks very promising though it could be overkill for you.